data/cfdm_test.pgdump.sql

```diff
@@ -26755,10 +26755,13 @@ COPY dimreporttype (reporttype_sk, rpt_tp, rpt_tp_desc, load_date, expire_date)
 --
 
 COPY dimyears (year_sk, year, load_date) FROM stdin;
-2040	2040	2014-09-03 11:27:56
-2003	2003	2014-09-03 11:27:56
-2052	2052	2014-09-03 11:27:56
-1999	1999	2014-09-03 11:27:56
+2040	2040	2014-09-03 11:27:56 \N
+2012    2012    2014-09-03 11:27:56 \N
+2013	2013	2014-09-03 11:27:56 \N
+2014    2014    2014-09-03 11:27:56 \N
+2008    2008    2014-09-03 11:27:56 \N
```

Comment:
Actually, don't spend time on the whitespace. I've located the relevant portions.

Just try to avoid a lot of whitespace changes mixed with real changes in the future. :)